demosymfonyform/src/Form/DemoConfigurationType.php

```diff
@@ -73,6 +74,20 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                         ],
                     ],
                 ]
-            );
+            )
+            ->add('translatable_formatted_text_area_type', TranslatableType::class, [
+                'label' => $this->trans('Translatable formatted text area type', 'Modules.DemoSymfonyForm.Admin'),
+                'help' => $this->trans('Throws error if length is > 10', 'Modules.DemoSymfonyForm.Admin'),
+                'type' => FormattedTextareaType::class,
+                'locales' => $this->locales,
```

Comment:
I wonder why is it required to set the `locales` option here, but it's not necessary for `TextareaType` or `TextInput`?
So is `FormattedTextareaType` a special case or is it that we forgot it for other types?

Reply:
Ok from what I see in `TranslatableType` it provides this option itself by default But nothing prevents from overriding it anyway Even though in this case I think it will have the same behaviour

Reply:
It seems you are right, I removed this locales as it's not really needed.